INSTALL

```diff
@@ -99,6 +99,15 @@
     $ nmake test
     $ nmake install
 
+ Note that in order to perform the install step above you need to have
+ appropriate permissions to write to the installation directory. For example,
+ in many cases on Unix, you may need to instead run the command like this:
+
+    $ sudo make install
```

Comment:
It might sensible be to add a warning here that it is not recommended to install to the default location on a linux/unix system, overwriting the preinstalled openssl version from the distributor. _(You made such a recommendation just recently on a GitHub issue, but I don't recall where it was)_

Also, you could add the recommendation to install to a writable location without `sudo` (e.g. below the home directory) using the `--prefix` configure option, if it's for testing purposes only.